docs/usage/validation_errors.md

```diff
@@ -792,13 +754,48 @@ class Model(BaseModel):
 
 try:
     Model(x='test')
-except ValidationError as exc_info:
-    assert exc_info.errors()[0]['type'] == 'int_parsing'
+except ValidationError as exc:
+    print(repr(exc.errors()[0]['type']))
+    #> 'int_parsing'
+```
+
+## `int_parsing_size`
+
+This error is raised when attempting to parse a python value from a string outside the maximum range of a Python `int`,
+or when trying to parse an integer from JSON that is outside the valid range of a 64 bit integer:
+
+```py
+import json
+
+from pydantic import BaseModel, ValidationError
+
+
+class Model(BaseModel):
+    x: int
+
+
+# from Python
+assert Model(x='1' * 4_300).x == int('1' * 4_300)  # OK
+
+too_long = '1' * 4_301
+try:
+    Model(x=too_long)
+except ValidationError as exc:
+    print(repr(exc.errors()[0]['type']))
+    #> 'int_parsing_size'
+
+# from JSON
+i64_max = 9_223_372_036_854_775_807
+try:
+    Model.model_validate_json(json.dumps(-i64_max * 2))
+except ValidationError as exc:
+    print(repr(exc.errors()[0]['type']))
+    #> 'dict_type'
```

Comment:
we are in the `int_parsing_size` section, but the error here is `dict_type`

Reply:
good catch, I'll fix it